$NetBSD: patch-make_common_MakeBase.gmk,v 1.3 2022/07/10 14:47:25 ryoon Exp $

* If both of hg and git are unavailable, 'gmake images' stalls.

--- make/common/MakeBase.gmk.orig	2022-04-23 02:23:55.000000000 +0000
+++ make/common/MakeBase.gmk
@@ -318,11 +318,7 @@ SCM_TIP_FILECMD := $(PRINTF) "$(SCM):%s"
 # Emit the scm:id pair to $@
 define GetSourceTips
 	$(CD) $(SRC_ROOT) ; \
-	if [ -d $(SCM_DIR) -a "$(SCM_VERSION)" != "" ] ; then \
-	  $(ID_COMMAND) >> $@ ; \
-	elif [ -f $(SCM_TIP_FILENAME) ] ; then \
-          $(SCM_TIP_FILECMD) >> $@ ; \
-	fi;
+        echo 'release' >> $@ ; \
 	$(PRINTF) "\n" >> $@
 endef
 
